tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -243,7 +243,7 @@ def fake_magpie_api_request(url, data):
 def test_harvesting_arxiv_workflow_rejected(
     mocked_api_request_beard_block, mocked_api_request_magpie,
     mocked_api_request_beard, mocked_download,
-    app, record_oai_arxiv_plots):
+    small_app, record_oai_arxiv_plots):
```

Comment:
Thanks! It could have been written in https://github.com/inspirehep/inspire-next/blob/master/tests/integration/conftest.py#L39